packages/xpetra/src/Export/Xpetra_TpetraExport_decl.hpp

```diff
@@ -208,6 +208,16 @@ toTpetra(const Export<LocalOrdinal, GlobalOrdinal, Node>& exp)
     return *tpetraExport.getTpetra_Export();
 }
 
+  template <class LocalOrdinal, class GlobalOrdinal, class Node>
+  const RCP< const Tpetra::Export< LocalOrdinal, GlobalOrdinal, Node > > toTpetra(const RCP< const Export< LocalOrdinal, GlobalOrdinal, Node > > &exportObj) {
```

Comment:
Are you sure that a `const` return type makes sense, why not just `RCP<const ...>`?

Reply:
This is similar to the previous `toTpetra` implementations.